### ISSUE TITLE
Please provide more details

### ISSUE THREAD

**Issue by SamP-dgt** [Crocoblock/jet-engine-sub-query#1]
Hi MjHead,

Your add-on seems very promising for what I am going to achieve (passing a parameter to a sub listing).

Would you please elaborate on its use? How are the new property fields supposed to be filled?

Many thanks

**Comment by drewbclark** [comment]
Yes I agree! This looks awesome just am confused on how to use it. Video Example or photo steps would be so helpful @MjHead @Crocoblock

**Comment by JuanCar29** [comment]
I have tried many things and no results. It would be nice if they documented the plugin.
